run.sh

```diff
@@ -14,5 +14,5 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
-
-python oneview_redfish_toolkit/app.py
+export PYTHONPATH=$PYTHONPATH:.
```

Comment:
I was looking at activate script it unsets any pythonpath variable. So chances are that 99.9% of time this will be empty as we a enforcing the use virtual env. The resulting path will be :. (NOTHING:.) I think this is an invalid path. I recommend this:
PYTHONPATH=${PYTHONPATH:+$PYTHONPATH:}.
If pythonpath is not empty add it to new pythonpath, otherwise set it only to .